README.md

```diff
@@ -68,37 +70,40 @@ We rely on different `django` and `mypy` versions:
 | 0.12.x       | old semantic analyzer (<0.711), dmypy support | 2.1.x | ^3.6
 
 
-## FAQ
+# FAQ
 
-### Is this an official Django project?
+### **Question:** Is this an official Django project?
 
-No, it is not. We are independent from Django at the moment.
+**Answer:** No, it is not. We are independent from Django at the moment.
 There's a [proposal](https://github.com/django/deps/pull/65) to merge our project into the Django itself.
 You can show your support by liking the PR.
 
-### Is it safe to use this in production?
+##
```

Comment:
why insert empty heading 2's ??

Reply:
To create a line between questions. It will be more readable for the user. Please  let me know if I need to make any changes there.